flink-kubernetes/src/main/java/org/apache/flink/kubernetes/utils/Constants.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.kubernetes.utils;
+
+/**
+ * Constants for kubernetes.
+ */
+public class Constants {
```

Comment:
I am not quite sure whether it is worth defining this class for only one constant variable, or we can define this variable directly in `KubernetesTaskExecutorRunner` class. If you prefer to a separate class, then it is better to rename the class as KubernetetsConstants to give a specific semantic.

Reply:
I will make this PR depends on #FLINK-10935. The class `Constants` will be created in that PR. We just add a new variable `ENV_FLINK_POD_NAME`. Does it make sense?

Reply:
Makes sense for me